### ISSUE TITLE
[Bug Report] The app crashes when playing the next song in my own list

### ISSUE THREAD

**Issue by fxzxmicah** [BobbyESP/Jetispot#25]
### Describe the bug

The app crashes when playing the next song in my own list.

### To Reproduce

1. Open my own playlist
2. Click the play button
3. Tap the next song button or wait for the next song to play automatically
4. App may crash

### Error reports & Screenshots

Application crashes directly, so there's no screenshot.

### Device info

0.1.7
Android 13

**Comment by SiddhantRakh** [comment]
It will happen if you don't have Spotify Premium. This app is intended to be used only if you have Spotify Premium.

**Comment by fxzxmicah** [comment]
> It will happen if you don't have Spotify Premium. This app is intended to be used only if you have Spotify Premium.

No, it's not. I can't even login if I don't have Spotify Premium.

**Comment by SiddhantRakh** [comment]
Ok sorry I was able to login without Premium on a previous build.
